shared/chat/conversation/messages/cards/team-journey/container.tsx

```diff
@@ -47,23 +48,27 @@ const TeamJourneyContainer = (props: Props) => {
 
   switch (props.message.cardType) {
     case RPCChatTypes.JourneycardType.welcome:
-      actions =
-        props.teamType === 'big'
-          ? [
-              {label: 'Browse channels', onClick: props.onBrowseChannels},
-              {label: 'Publish team on your profile', onClick: props.onPublishTeam},
-            ]
-          : [{label: 'Publish team on your profile', onClick: props.onPublishTeam}]
+      image = 'icon-illustration-welcome-96'
+      actions = props.teamType === 'big' ? [{label: 'Browse channels', onClick: props.onBrowseChannels}] : []
```

Comment:
this thingy would be more legible to me as
```
actions = [
   ...((props. ... big') ? [thing] : []),
   ...((props.canS...) ? [thing] : []),
]```
than a ternary followed by an if

Reply:
it was messy. What about this?